### PR TITLE
Feature/update food component

### PR DESCRIPTION
**Pull request by yannhn**

### CODE REVIEW

File: src/components/FoodCard/index.js [hunk 1]
```diff
@@ -43,21 +29,15 @@ export default function FoodCard() {
...
+				))}
 			</FoodCardContainer>
-			{showForm && (
-				<InputFood
-					addNewFoodItem={addNewFoodItem}
-					setSubmittedFoodInput={setSubmittedFoodInput}
-					setSubmittedTimeInput={setSubmittedTimeInput}
-				></InputFood>
-			)}
+			{showForm && <InputFood addNewFoodItem={addNewFoodItem}></InputFood>}
```
Comment (macfrei) [inline src/components/FoodCard/index.js:40]:
```suggestion
			{showForm && <InputFood addNewFoodItem={addNewFoodItem} />}
```

File: src/components/FoodCard/index.js [hunk 2]
```diff
@@ -43,21 +29,15 @@ export default function FoodCard() {
...
-						<p>What: {submittedFoodInput} gram</p>
-						<p>When: {submittedTimeInput}</p>
-					</FoodInfo>
-					<FoodCounter>{todayAmount} / goal</FoodCounter>
-				</FoodInfoContainer>
+				{foodItems.map(item => (
+					<FoodInfoContainer key={item.id}>
+						<p>How much: {item.amount} gram</p>
+						<p>At: {item.time}</p>
+						<hr></hr>
```
Comment (macfrei) [inline src/components/FoodCard/index.js:36]: I would recommend to not use the `hr` element but do thy styling via CSS. 🙂 It's my personal preference not to add unnecessary HTML elements.

**Review by macfrei — APPROVED** [review]
Good improvements. Just added two comments you might want to consider but it's basically just nit-picking. 🙂

**Comment by yannhn** [comment]
Thanks! I already implemented the feedback

**Pull request merged**